src/Adapter/MSTest.TestAdapter/PublicAPI/PublicAPI.Unshipped.txt

```diff
@@ -1 +1 @@
-﻿
\ No newline at end of file
+﻿Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter.ObjectModel.TestMethod.TestIdGenerationStrategy.get -> Microsoft.VisualStudio.TestTools.UnitTesting.TestIdGenerationStrategy
```

Comment:
move to shipped?

Reply:
I was thinking about moving all APIs to shipped either right before release or right after but in this way it's easier (IMO) to understand if that's an API I can still change because it's not yet published.

Reply:
By experience we forgot to do that, but it's fine the same.

Reply:
Will keep it as unshipped for now and if I see the same problem of "we forgot to move it", I will revise and use only shipped file. Thanks!